docs/dom-testing-library/api-events.mdx

```diff
@@ -138,23 +138,31 @@ test that a callback passed to the function was called, or what it was called
 when the event that **should** trigger the callback function does trigger the
 bound callback.
 
-<!--DOCUSAURUS_CODE_TABS-->
-
-<!--React-->
-
-```jsx
-import { render, screen } from '@testing-library/react'
-
-const Button = ({ onClick, children }) => (
-  <button onClick={onClick}>{children}</button>
-)
-
-test('calls onClick prop when clicked', () => {
-  const handleClick = jest.fn()
-  render(<Button onClick={handleClick}>Click Me</Button>)
-  fireEvent.click(screen.getByText(/click me/i))
-  expect(handleClick).toHaveBeenCalledTimes(1)
-})
-```
+import Tabs from '@theme/Tabs';
```

Comment:
Nit: maybe we should put imports at the top of the page?

Reply:
Definitely, done.